pkg/pillar/cmd/zedrouter/probe.go

```diff
@@ -141,6 +147,10 @@ func niProbingUpdatePort(ctx *zedrouterContext, port types.NetworkPortStatus,
 		netstatus.PInfo[port.IfName] = info
 		log.Infof("niProbingUpdatePort: %s assigned new %s, info len %d, isFree %v\n",
 			netstatus.BridgeName, port.IfName, len(netstatus.PInfo), info.IsFree)
+		if !ipAddrIsValid(info.LocalAddr) {
+			info.GatewayUP = false
+			info.RemoteHostUP = false
+		}
```

Comment:
I think these variables have a default value of false. Don't need explicit assignment, unless the intention is to make the assignment look explicit.

Reply:
this is not default by that time, since the initailization above that lines has assign them to 'true' in general, I wish there is 'C' language mechanism can do a = b ? x : y.

Reply:
Ok. got it.